forms-flow-web/src/components/Draft/Edit.js

```diff
@@ -143,6 +140,18 @@ const View = React.memo((props) => {
 
   return (
     <div className="container overflow-y-auto">
+      {
+        <>
+          <span className="pr-2  mr-2 d-flex justify-content-end align-items-center">
```

Comment:
@sreehari-aot For Anonymous form submission we don't need to show draft indicatiin